feature-libs/cart/quick-order/assets/translations/en/quick-order.i18n.ts

```diff
@@ -12,12 +12,14 @@ export const quickOrderCartForm = {
 
 export const quickOrderForm = {
   placeholder: 'Enter Product SKU',
+  listLimitReached: 'The maximum products level has been reached',
 };
 
 export const quickOrderList = {
-  addProducts: 'Add Products/Skus',
-  emptyList: 'Empty list',
   addToCart: 'Add to cart',
+  emptyList: 'Empty list',
+  header: 'Add Products/Skus',
+  subHeader: `You can add up to {{ limit }} valid SKU's below and add to cart`,
```

Comment:
Similar to the other label, but not sure what to change here. Ask Matthew's opinion or Bill

Reply:
not sure what is wrong here

Reply:
verified wireframes 👍